control/matlab/__init__.py

```diff
@@ -84,6 +84,7 @@
 from ..rlocus import rlocus
 from ..dtime import c2d
 from ..sisotool import sisotool
+from ..stochsys import *
```

Comment:
Can we make these explicit imports please?

Reply:
I think the two functions that we want (since they are present in MATLAB) are lqe and dlqe.  The other functions are unique to python-control.